http-api/src/custom_axum.rs

```diff
@@ -0,0 +1,57 @@
+use axum::extract::path::ErrorKind;
```

Comment:
If this is an "axum related" file, maybe we can just call it `axum.rs`

Reply:
Then I have to use `mod axum` in lib.rs which interferes with the extern crate of `axum` itself.

For now I changed filename to axum.rs and import it with:
```
#[path = "axum.rs"]
mod custom_axum;
```

Reply:
Ah gotcha yes. I guess by convention these are called "extra" modules, eg. we have a `git_extra` module in radicle link, so maybe `axum_extra` would make sense here?